File: RemsNG/RemsNG-UI/src/app/role/models/role.model.ts
export class RoleModel {
    id: string = '';
    roleName: string = 'Nil';
    roleStatus: string = '';
    domainId: string = '';
    isErrMsg: boolean = false;
    errMsg: string = '';
    eventType: string = '';
    errClass: string[] = [];
    isLoading: boolean = false;
    domainName: string = '';
}

File: RemsNG.Common/Models/RoleModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace RemsNG.Common.Models
{
    public class RoleModel
    {
        public Guid Id { get; set; }
        public string RoleName { get; set; }
        public Guid DomainId { get; set; }
        public string RoleStatus { get; set; }
        public string domainName { get; set; }

        public virtual LcdaModel Domain { get; set; }
    }
}
